README.md

```diff
@@ -10,8 +10,9 @@ This JavaScript library is part of the [Microsoft Teams developer platform](http
 ## Getting Started
 
 1. Clone the repo
-2. `yarn build` from repo root
-3. to run Unit test `yarn test`
+2. `yarn install` from repo root
```

Comment:
Can you please make this change in our other packages/teams-js/README.md too?

Reply:
The change is already in that README file, but it was missing in this one. Would probably make sense to only have one README file and then copy it around during the build. Not sure how you all track issues but I guess this would be a good one to throw on the pile?